xt/01-html-checks.t

```diff
@@ -5,8 +5,22 @@ use warnings;
 
 use File::Find;
 use Test::More;
-eval "use  HTML::Lint::Pluggable";
-plan skip_all => "HTML::Lint::Pluggable not available" if $@;
+
+use LedgerSMB::Sysconfig;
+
+use Hash::Merge qw( merge );
+use Scalar::Util;
+
+use Template;
+use Template::Config;
+use Template::Stash;
+use Template::VMethods;
+
+use WebService::Validator::HTML::W3C;
+
+$Template::Stash::PRIVATE = undef;  # Make sure we can access variables with a leading underscore
+
+my $validator_uri = $ENV{'VALIDATOR_URI'} // 'http://localhost:45000/w3c-validator/check';
```

Comment:
I disagree here: this means the tests will fail when the validator isn't set up. Instead of using a default, we should skip.